### DISCUSSION TITLE
How to avoid DBTIMEZONE warning?

### DISCUSSION THREAD

**Discussion by denissun** [godror/godror#276] [General]
My command line Go program receives the following warning:

godror WARNING: discrepancy between DBTIMEZONE ("+00:00"=0) and SYSTIMESTAMP ("-05:00"=-500) - set connection timezone, see https://github.com/godror/godror/blob/master/doc/timezone.md

I checked the link, but still don't know how to avoid it? I played with CommonParams Timezone, but not sure if set something there would help or not?

**Comment by tgulacsi** [comment]
I've extended the documentation - TL;DR add a `timezone="+0500"` param to your connection string.

**Comment by denissun** [comment]
I figured out with the following code

tz, _ := time.LoadLocation("America/New_York")

	return godror.ConnectionParams{
		StandaloneConnection: true,
		CommonParams: dsn.CommonParams{
			Username:      p_username,
			Password:      dsn.NewPassword(p_password),
			ConnectString: connString,
			Timezone:      tz,
		},


If I do:  Timezone :  "-0500"  not working

**Reply by tgulacsi** [reply]
If it works with "America/New_York", then use that.
